opentelemetry-api/tests/context/test_contextvars_context.py

```diff
@@ -13,21 +13,20 @@
 # limitations under the License.
 
 import unittest
+from sys import version_info
 from unittest.mock import patch
 
 from opentelemetry import context
 
 from .base_context import ContextTestCases
 
-try:
-    import contextvars  # pylint: disable=unused-import
-
-    from opentelemetry.context.contextvars_context import (
-        ContextVarsRuntimeContext,
-    )
-except ImportError:
+if version_info.minor < 7:
```

Comment:
why is this change here?

Reply:
The current approach to find out the version is to try to import `contextvars`. If this fails we know we the running Python version is < 3.7. This introduces an unused import symbol (`contextvars`) when we can just simply query `version_info.minor` to find out the same thing.